src/test/java/bio/terra/common/auth/AuthService.java

```diff
@@ -68,6 +75,13 @@ public String getAuthToken(String userEmail) {
     return userTokens.get(userEmail).getAccessToken();
   }
 
+  public String getPetAccountAuthToken(String userEmail) {
+    if (!petAccountTokens.containsKey(userEmail)) {
+      petAccountTokens.put(userEmail, makePetAccountToken(userEmail));
+    }
+    return petAccountTokens.get(userEmail);
```

Comment:
you can use `computeIfAbsent()` here
```suggestion
    return petAccountTokens.computeIfAbsent(userEmail, this:: makePetAccountToken);
```

Reply:
good call!